source/GameData.cpp

```diff
@@ -1387,10 +1405,161 @@ void GameData::PrintWeaponTable()
 		cout << hull << '\t';
 		double hitforce = outfit.HitForce() * 60. / outfit.Reload();
 		cout << hitforce << '\t';
+        double heatdmg = outfit.HeatDamage() * 60. / outfit.Reload();
+        cout << heatdmg << '\t';
+        double slow = outfit.SlowingDamage() * 60. / outfit.Reload();
+        cout << slow << '\t';
+        double disruption = outfit.DisruptionDamage() * 60. / outfit.Reload();
+        cout << disruption << '\t';
+        double ion = outfit.IonDamage() * 60. / outfit.Reload();
+        cout << ion << '\t';
```

Comment:
We have other damage types that also aren't printed. But like, I don't know whether we should print every one of them either so....

Reply:
Those aren't used in the base game, but we don't know if plugins use them at all. I think it would be fair to put them in.